### DISCUSSION TITLE
Circuit Board

### DISCUSSION THREAD

**Discussion by shanekuz** [wayne-love/sn_esp32#6] [General]
What is the theory with the 330K resistors in line with the RX TX?  I'm designing a board once complete will put up a link so people can order it from the board printing site with links to the parts.

I don't have any resistors at the moment and it works fine.

**Comment by wayne-love** [comment]
Safety, as all pins are programmable it is possible to have a TX <--> TX connection and this is "bad".  The resistor limits the possible current (and acts as a fuse) + it also helps eliminate ringing on the circuit. I am so out of my depth with this it is not funny but have a look here (https://resources.pcb.cadence.com/blog/termination-resistors-in-pcb-design).  

I came across the concept when feeling out the pins on the Wifly, they have a 330-ohm resistor on the RX pin.  As pins on the esp32 are interchangeable, the common practice seems to be to put a resistor on both.
